packages/grafana-schema/src/schema/dashboard/v2alpha0/dashboard.gen.ts

```diff
@@ -145,6 +145,17 @@ export const defaultDataTransformerConfig = (): DataTransformerConfig => ({
 	options: {},
 });
 
+export interface DataLink {
```

Comment:
I'm pretty sure that these fields are all we need to persist in db. The old schema's DataLink has https://github.com/grafana/grafana/blob/002f872ce17fe9d93a949008223be9170983c333/packages/grafana-data/src/types/dataLink.ts#L32 origin and meta fields, but these are not persisted and are setup during runtime and are relevant only for Explore.